internal/serve/httphandler/forgot_password_handler_test.go

```diff
@@ -380,6 +402,35 @@ func Test_ForgotPasswordHandler(t *testing.T) {
 		assert.JSONEq(t, wantsBody, string(respBody))
 	})
 
+	t.Run("Should return http status 400 when attempting to validate reCAPTCHA token after failing to find user by email", func(t *testing.T) {
+		requestBody := `
+		{ 
+			"email": "valid@email.com"
```

Comment:
Would this be a better naming? WDYT?
```suggestion
			"email": "not-found@email.com"
```